src/phase_space_layouts/out_channel/flat_phase_space.jl

```diff
@@ -0,0 +1,149 @@
+# here we implement RAMBO
+
+struct FlatPhaseSpaceLayout{INPSL} <: QEDcore.AbstractOutPhaseSpaceLayout{INPSL}
+    in_psl::INPSL
+end
+
+function QEDcore.phase_space_dimension(
+    proc::AbstractProcessDefinition,
+    model::AbstractModelDefinition,
+    psl::FlatPhaseSpaceLayout,
+)
+    return 4 * number_outgoing_particles(proc)
+end
+
+QEDcore.in_phase_space_layout(psl::FlatPhaseSpaceLayout) = psl.in_psl
+
+function QEDcore._build_momenta(
+    proc::AbstractProcessDefinition,
+    model::AbstractModelDefinition,
+    in_moms::Tuple,
+    psl::FlatPhaseSpaceLayout,
+    out_coords::Tuple,
+)
+    # preparing inchannel
+    Ptot = sum(in_moms)
+    boost_from_rest = inv(_unsafe_rest_boost(Ptot))
+
+    sqrt_s = Base.sqrt_llvm(Ptot * Ptot)
```

Comment:
What's the reason for using `Base.sqrt_llvm`? It's an internal function so we should probably not rely on it.
It actually seems pretty unsafe, because 
```
julia> Base.sqrt_llvm(64)
2202260217784172544
```
is probably not what you would expect.

Reply:
Sorry, this was a left-over from the development phase. I replaced it with the standard `sqrt`.